debugger/gdb-sgx-plugin/thread.py

```diff
@@ -0,0 +1,326 @@
+import gdb
+import math
+import tempfile
+from typing import List
+
+DEBUG = False
+
+INSTANCE_DEBUG_EXT = None
+
+class ThreadCloneBreakpoint(gdb.Breakpoint):
+    def __init__(self):
+        """
+        THIS BREAKPOINT IS DISABLED AS FUNCTION CALL MAY NOT BE RELIABLE
+
+        Set a breakpoint @ kernel/syscall.c::_SYS_myst_clone
+        Choosing this location because this is where the parent process/thread creates new process/thread
+        We can get the parent pid/tid from here
+        """
+        super(ThreadCloneBreakpoint, self).__init__('kernel/syscall.c:4094', internal=False)
+
+    def stop(self):
+        # Get Parent thread info
+        ppid = int(gdb.parse_and_eval('myst_thread_self()->process->ppid'))
```

Comment:
oegdb does not guarantee that calling enclave functions from the debugger would work.
Therefore, we cannot rely on calling `myst_thread_self()`.

With lldb, enclave function calls are not supported at all.

Reply:
Will remove this usage